docs/source/guide/qse.md

```diff
@@ -0,0 +1,27 @@
+# Quantum Subspace Expansion
+
+Quantum Subspace Expansion (QSE) is an error mitigation technique in which
+we define a small subspace around the output state, then search this subspace for the state with the least amount of error. The subspace is defined by a set of operators that act on the output state called the check operators. Subspace expansion can be used in conjunction with a stabilizer code, in which case the stabilizers, or a subset of them, can be used as the check operators. If the full set of stabilizers is taken as the check operator, the net effect of subspace expansion is to project the output state onto the codeword subspace. 
```

Comment:
One question that came to mind that may be helpful in clarifying this section:

> How do we calculate error if we don't know what output state we are looking for?

---

I think the last sentence in this highlight can be removed. I don't think it needs to be front and center, and is in the theory section as well. WDYT?

Reply:
I clarified that it's the same procedure regardless of the output state